src/mmw/js/src/app.js

```diff
@@ -15,17 +15,51 @@ var App = new Marionette.Application({
 
         // This view is intentionally not attached to any region.
         this._mapView = new views.MapView({
-            model: this.map
+            model: this.map,
+            el: '#map'
         });
 
         this.rootView = new views.RootView();
         this.user = new userModels.UserModel({});
 
-        var header = new views.HeaderView({
+        this.header = new views.HeaderView({
             el: 'header',
             model: this.user
         });
-        header.render();
+        this.header.render();
+
+        // Not set until modeling/controllers.js creates a
+        // new project.
+        this.currProject = null;
+    },
+
+    // Should be called after each unit test.
+    reset: function() {
```

Comment:
I feel like this should be part of the test code and not part of the App. We have no need to reset the app in non-test code do we?